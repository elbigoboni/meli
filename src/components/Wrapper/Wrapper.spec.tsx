import { render } from "@testing-library/react";
import React from "react";
import { MemoryRouter } from "react-router-dom";
import { Wrapper } from ".";

describe("<Wrapper />", () => {
  describe("Render", () => {
    test("should render correctly", () => {
      const { container } = render(<Wrapper />, { wrapper: MemoryRouter });
      expect(container).toMatchSnapshot();
    });
  });
});
