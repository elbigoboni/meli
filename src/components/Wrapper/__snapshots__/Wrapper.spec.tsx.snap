// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Wrapper /> Render should render correctly 1`] = `
<div>
  <header
    class="sc-bdfBwQ jjqhxy"
  />
</div>
`;
